ajax/libs/hyperHtml/package.json

```diff
@@ -1,33 +1,33 @@
 {
-    "name": "hyperhtml",
-    "filename": "min.min.js",
-    "description": "A Fast & Light Virtual DOM Alternative",
-    "license": "ISC",
-    "author": "Andrea Giammarchi",
-    "repository": {
-      "type": "git",
-      "url": "git+https://github.com/WebReflection/hyperhtml.git"
-    },
-    "keywords": [
-      "dom",
-      "diff",
-      "performance",
-      "template",
-      "literals",
-      "lightweight",
-      "fast",
-      "react",
-      "virtual",
-      "lit-html",
-      "alternative"
-    ],
-    "npmName": "hyperhtml",
-    "npmFileMap": [
-     {
-       "basePath": "dist",
-       "files": [
-         "**/*"
-       ]
-     }
+  "name": "hyperHtml",
+  "filename": "min.min.js",
```

Comment:
oh gosh ... really? it's just min.js in the repo :smile:

Reply:
@WebReflection lol I kinda just guessed what they meant by "main file" and the convention said to use filename.min.js so.... ya I dunno if this is right but you'd probs know best lol ! so I change this to min.js ???

Reply:
thanks @WebReflection I'm still confused a bit so do I just take the filename out ??

Reply:
I've no idea what this bot expect, sorry :cry:

Reply:
haha its all good ! :) 🏆  Ill wait for one of the reviews to check it out, I guess somethings wrong with my formatting ?? tabs vs spaces 👾  things lol I used a "Linter" but that still doesn't seem to do the trick and I have no idea how to access the cdnjs tool they mention.